api/opensource/mfx_dispatch/src/mfx_dispatcher.cpp

```diff
@@ -25,8 +25,13 @@
 #include <assert.h>
 
 #include <string.h>
+#if defined(_WIN32) || defined(_WIN64)
+    #include <windows.h>
+    #pragma warning(disable:4355)
+#else
     #include <dlfcn.h>
     #include <iostream>
+#endif // defined(_WIN32) || defined(_WIN64)
```

Comment:
hm. is that the only place which miss Windows support? Did you check that Windows dll loading code is actually available?